deepspeed/linear/config.py

```diff
@@ -0,0 +1,21 @@
+# Copyright (c) Microsoft Corporation.
+# SPDX-License-Identifier: Apache-2.0
+
+# DeepSpeed Team
+
+from dataclasses import dataclass
+
+
+@dataclass
+class LoRAConfig:
+    lora_r: int = 64
+    lora_alpha: float = 16
+    base_weight_sharding: int = 1
+
+
+@dataclass
+class QuantizationConfig:
```

Comment:
Is it possible to have a more descriptive name instead of `QuantizationConfig`? Afterall, LoRA is a form of quantization, so why is it treated differently?